reference/docs-conceptual/learn/experimental-features.md

```diff
@@ -88,6 +89,19 @@ In this example, a job is started and a breakpoint is set to break when the `Set
 run. The runspace is stored in a variable and passed to the `Get-PSBreakPoint` command with the
 **Runspace** parameter. You can then inspect the breakpoint in the `$breakpoint` variable.
 
+## PSAMSIMethodInvocationLogging
+
+The Windows Antimalware Scan Interface (AMSI) is an API that allows application actions to be passed
+to an antimalware scanner, such as Windows Defender, to be scanned for malicious payloads. Beginning
+with PowerShell 5.1, PowerShell running on Windows 10 (and higher) passes all script blocks to AMSI.
+
+This experimental feature extends the data that is sent to AMSI for inspection. With this feature
+enabled, PowerShell now includes any invocation of a .NET method member.
```

Comment:
Does it include _any_ invocation (i.e. opt-in) or _all_ invocations by default?

Reply:
See updated changes